robotsim.py

```diff
@@ -254,6 +258,45 @@ def getColor(self):
             return map.tiles[row][col].color
         return 'white'
 
+    def sendMessageRescueBase(self, path = None):
+        row = self.row
+        col = self.col
+        if map.tiles[row][col].envType == "people":
```

Comment:
we should first verify the path and then eliminate the people; if the path is wrong, the people should stay there

Reply:
after some discussion we can do this:
-verify if code is correct,
->if not correct, people stay, path is not verified and points are eliminated
->if correct, people leave and we verify the extra points of the path

This will get rid of the issue of having infinite points with the path